datahub-web-react/src/graphql/mutations.graphql

```diff
@@ -123,3 +123,7 @@ mutation createPost($input: CreatePostInput!) {
 mutation updateLineage($input: UpdateLineageInput!) {
     updateLineage(input: $input)
 }
+
+mutation updateEmbed($input: UpdateEmbedInput!) {
+    updateEmbed(input: $input)
+}
```

Comment:
it doesn't look like we're going to allow users to edit this through the UI as of now, right?

Reply:
Nope - Only via API if they'd like! Eventually I'd like to add a form to do it via UI. So owners could provide something custom post-ingestion